assets/css/src/components/_wp-list-table-markup-status-icon.scss

```
@@ -0,0 +1,28 @@
+// Add markup status icon next to the dropdown.
+.wp-list-table.table-view-list > tbody > tr {
+
+	.amp-validation-error-status-dropdown {
+		align-items: center;
+		display: flex;
+	}
+
+	.amp-validation-error-status-dropdown::after {
+		background: #d3e5e5 url("../images/amp-delete.svg") no-repeat center;
+		background-size: 20px auto;
+		border-radius: 5px;
+		content: "";
+		display: inline-block;
+		height: 20px;
+		width: 20px;
+	}
+
+	&.kept .amp-validation-error-status-dropdown::after {
+		background-color: #f7ded4;
+		background-image: url("../images/amp-alert.svg");
+		background-size: 18px auto;
+	}
+
+	.amp-validation-error-status { // SELECT tag
```

Why not simply add the `select` tag?

```suggestion
	select.amp-validation-error-status {
```